File: 프로그래머스/0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string my_string) {
    int answer = 0;
    string s = "";
    for(int i = 0; i<my_string.size(); i++){
        if(my_string[i] <= 57){
            s += my_string[i];
        }else if(s.size()>0){
            answer += stoi(s);
            s = "";
        }
    }
    if(s.size()>0){
        answer += stoi(s);
        s = "";
    }
    return answer;
}

File: 프로그래머스/0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.py
def solution(my_string):
    answer = 0
    temp = ""
    for i in my_string:
        if i.isdigit():
            temp += i;
        else:
            temp += ' ';
            
    temp = temp.split()
    
    for i in temp:
        answer += int(i)
    
    return answer